hangman.js

```diff
@@ -1 +1,28 @@
-console.log("welcome to hangman");
\ No newline at end of file
+var guest= prompt('Please enter your Nickname', '');
+if ( (guest=='') || (guest==null) )
+{
+ guest="";
+}
```

Comment:
Thanks Joe, I will look over and make corrections this weekend :)

On Fri, Feb 19, 2016 at 1:11 PM, Joe Glatman Zaretsky <
notifications@github.com> wrote:

> In hangman.js
> <https://github.com/ga-wdi-exercises/hangman-lab/pull/20#discussion_r53495557>
> :
>
> > \ No newline at end of file
> > +var guest= prompt('Please enter your Nickname', '');
> > +if ( (guest=='') || (guest==null) )
> > +{
> > + guest="";
> > +}
>
> this if statement probably isnt necessary. If the prompt is left blank, it
> will return an empty string regardless.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ga-wdi-exercises/hangman-lab/pull/20/files#r53495557>.
>



-- 
*LOVE *

*NOTE: This e-mail transmission, including any attachments, is intended
only for the named recipient(s) and may contain information that is
privileged, confidential and/or exempt from disclosure under applicable
law. If you have received this email by intent or accident, you are
forbidden to disclose any information attached or written with anyone
outside of whom this email was intended for. If you have received this
transmission in error, or are not the named recipient(s), please notify
Kaira Johnson <kjohnson1913@gmail.com> immediately by return e-mail and
permanently delete this transmission, including any attachments.*